test/maneuversbuilder.cc

```diff
@@ -1835,9 +1835,9 @@ TEST(Maneuversbuilder, TestPathSlightRightXSlightLeftIsIntersectingForwardEdge)
   node = path.add_node();
   edge = node->mutable_edge();
   PopulateEdge(edge, {{"Horace Greeley Road", 0}}, 0.205258, 30.000000,
-               valhalla::RoadClass::kResidential, 35, 19, 6, 12, TripLeg_Traversability_kBoth, 0, 0,
+               valhalla::RoadClass::kResidential, 45, 19, 6, 12, TripLeg_Traversability_kBoth, 0, 0,
                0, 0, 0, 0, 0, 0, 0, 0, {}, {}, {}, {}, TripLeg_TravelMode_kDrive);
-  PopulateIntersectingEdge(node->add_intersecting_edge(), 355, 0, 0, TripLeg_Traversability_kBoth,
+  PopulateIntersectingEdge(node->add_intersecting_edge(), 3, 0, 0, TripLeg_Traversability_kBoth,
```

Comment:
Why did these change? Was it required to make the test continue to pass?

Reply:
The logic to determine if the current edge is the straightest was not correct - the logic and padding threshold is now based on the delta from straight - not just the delta between the current edge and intersecting edge. Therefore, this test needed to be updated to validate the intersecting forward edge